plone/app/linkintegrity/browser/info.py

```diff
@@ -101,7 +102,23 @@ def get_breaches(self, items=None):
                         # This check is necessary since there can be multiple
                         # sources for a breach
                         results.remove(result)
-        return results
+
+        # De-duplicate targets * sources
+        uid_target = {}
+        uid_sources = defaultdict(list)
+        for result in results:
+            target_uid = result["target"]["uid"]
+            uid_target[target_uid] = result["target"]
+            sources = uid_sources[target_uid]
+            for source in result["sources"]:
+                if source not in sources:
+                    sources.append(source)
+
+        # List of breaches
+        return [
```

Comment:
@jaroel @mauritsvanrees I guess this is a breaking change in plone.app.querystring, right? So we have to do a major version dump and include it (only) in Plone 6.1, am I correct?

Reply:
Nah, this doesn't change the shape - just de-duplication.
I'll add a test to make this more visible, fair enough.